metrics/src/test/java/io/opencensus/metrics/DistributionTest.java

```diff
@@ -63,18 +68,16 @@ public void createAndGet_Distribution() {
     Exemplar exemplar = Exemplar.create(15.0, TIMESTAMP_1, ATTACHMENTS);
     List<Double> bucketBounds = Arrays.asList(-1.0, 0.0, 1.0);
     List<Bucket> buckets =
-        Arrays.asList(Bucket.create(3), Bucket.create(1), Bucket.create(2), Bucket.create(4));
-    Distribution distribution =
-        Distribution.create(
-            6.6, 10, 678.54, bucketBounds, buckets, Collections.singletonList(exemplar));
+        Arrays.asList(
+            Bucket.create(3), Bucket.create(1), Bucket.create(2), Bucket.create(4, exemplar));
+    Distribution distribution = Distribution.create(6.6, 10, 678.54, bucketBounds, buckets);
     assertThat(distribution.getMean()).isEqualTo(6.6);
     assertThat(distribution.getCount()).isEqualTo(10);
     assertThat(distribution.getSumOfSquaredDeviations()).isEqualTo(678.54);
     assertThat(distribution.getBucketBoundaries())
         .containsExactlyElementsIn(bucketBounds)
         .inOrder();
     assertThat(distribution.getBuckets()).containsExactlyElementsIn(buckets).inOrder();
-    assertThat(distribution.getExemplars()).containsExactly(exemplar);
```

Comment:
Should you add a test to check if exemplar is present in the bucket?

Reply:
SG, done.